inductiva/molecules/simulators/gromacs.py

```diff
@@ -53,3 +53,18 @@ def run_async(
         return super().run_async(input_dir,
                                  method=method_name,
                                  user_flags=gromacs_flags)
+
+
+class GROMACSCommand(Command):
+    """Class for construction of GROMACS commands."""
+
+    def __init__(self, method_name: str, **gromacs_flags: Optional[str]):
+        """Command constructor.
+
+        Args:
+            method_name: The name of the method to be executed in GROMAS.
+            **gromacs_flags: Additional keyword arguments to be passed 
+            to the simulation API method.
+        """
+        all_args = {"method_name": method_name, **gromacs_flags}
```

Comment:
In principle, you always have a `method_name` and may or may not have some flags. But the method_name is fixed, since it is calling a specific command in the executer side.

Reply:
Technically, we didn't need to also add a GromacsCommand on the backend ;)

Reply:
Ok, so your suggestion is to just keep a commands class, right?

Reply:
For now, yes! But I think we need to discuss if it is even necessary.

Reply:
I was having the same doubt! Because it is just a dictionary, but having the command classe will make the user do the proper syntax of the command and define a name for the command and other parameters we may want to pass as arguments.